packages-exp/app-check-exp/src/internal-api.test.ts

```diff
@@ -239,6 +236,7 @@ describe('internal api', () => {
     });
 
     it('ignores listeners that throw', async () => {
+      stub(console, 'error');
```

Comment:
Is it to just suppress error messages during testing? If so, can you please add comments to clarify?

Reply:
Yes, done.